### DISCUSSION TITLE
Yandex translation and dubbing

### DISCUSSION THREAD

**Discussion by AlexN1ght** [ThioJoe/Auto-Synced-Translated-Dubs#12] [General]
What do you think about Yandex Browser ability to translate youtube videos and even streams? It is just like Joe described in the end of the video but it is no longer a dream. Unfortunately it only translates to Russian, but the tecnology is there!
For reference: https://medium.com/yandex/yandex-browser-live-stream-translation-principles-and-differences-to-on-demand-video-dubbing-df9c131eb5b0

**Comment by Pythone** [comment]
This project should not depend on Yandex or any other Russian site
